api/src/main/java/science/aist/seshat/FactoryLoader.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020 the original author or authors.
```

Comment:
probably time to update the template. We somehow survived 2020

Reply:
Good point, updated it to 2021 for the one file.